common/ethnicities/wc_ethnicities_dwarven_rework.txt

```diff
@@ -0,0 +1,524 @@
+﻿# Warcraft
+###Skin colors
+@human_skin_color_x_min = 0.15
+@human_skin_color_x_max = 0.2
+
+###Hair colors
+@blonde_color_x_min = 0.2
+@blonde_color_x_max = 0.25
+@blonde_color_y_min = 0.15
+@blonde_color_y_max = 0.62
+
+@brown_color_x_min = 0.08
+@brown_color_x_max = 0.2
+@brown_color_y_min = 0.86
+@brown_color_y_max = 0.95
+
+@red_color_x_min = 0.05
+@red_color_x_max = @brown_color_x_min
+@red_color_y_min = 0.53
+@red_color_y_max = 0.65
+
+@white_color_y_min = 0.05
+@white_color_y_max = 0.05
+
+@black_color_y_min = 1
+@black_color_y_max = 1
+
+@tan_color_y_min = 0.3
+@tan_color_y_max = 0.4
+
+###Eye colors
+@blue_eye_color_x_min = 0.6
+@blue_eye_color_x_max = 0.7
+
+@arcane_color_y_min = 0.34
+@arcane_color_y_max = 0.53
+
+@blue_color_x_min = @blue_eye_color_x_min
+@blue_color_x_max = 0.83
+
+@ash_color_x_min = 0.72
+@ash_color_x_max = 0.75
+@ash_color_y_min = 0.2
+@ash_color_y_max = 0.25
+
+dwarven_rework = {
+	template = "caucasian"
+	
+	skin_color = {
+		1 = { 0.047 0.082 0.122 0.263 }
+	}
+	eye_color = {
+		1 = { 0 0.300 0.757 0.600 } #Most colors
+		1 = { 0.914 0.408 1 0.600 } #Pink
+	}
```

Comment:
I would expect dwarves to be the same eye colors as humans.
I'm not aware of lore characters with bright-colored eyes, so I assume it's player customization from the WoW.

Reply:
Yes indeed, I took the colors from WoW for the most part. I'll try to look for a better source.

Reply:
> Yes indeed, I took the colors from WoW for the most part. I'll try to look for a better source.

I would give them the same eye colors humans have.

Reply:
> 
> 
> > Yes indeed, I took the colors from WoW for the most part. I'll try to look for a better source.
> 
> I would give them the same eye colors humans have.

Changed this in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/200/commits/344a4ddb36c71ee4b4699e9361c6ac2cd533a9ee.